src/handler/mod.rs

```diff
@@ -468,6 +485,27 @@ impl Stream for Handler {
                     HandlerMessage::DisposeContext(ctx) => {
                         pin.browser_contexts.remove(&ctx);
                     }
+                    HandlerMessage::AddEventListener(req) => {
+                        pin.event_listeners.add_listener(req);
+                    }
+                    HandlerMessage::GetPage(target_id, tx) => {
+                        let target = pin
```

Comment:
`targets` is a `HashMap` with the `TargetId` as key so there is no need to iterate, `self.targets.get_mut` is enough